setup.py

```diff
@@ -0,0 +1,30 @@
+# ----------------------------------------------------------------------------
+# Copyright (c) 2018-2019, QIIME 2 development team.
+#
+# Distributed under the terms of the Modified BSD License.
+#
+# The full license is in the file LICENSE, distributed with this software.
+# ----------------------------------------------------------------------------
+
+from setuptools import setup, find_packages
+import versioneer
+
+setup(
+    name='q2-diversity-lib',
+    version=versioneer.get_version(),
+    cmdclass=versioneer.get_cmdclass(),
+    packages=find_packages(),
+    author="QIIME 2 development team",
```

Comment:
You can put your name here as the primary package author, if you wish.

Reply:
✓